preserve_model.py

```
@@ -47,6 +60,58 @@ def load_dotenv(*_args, **_kwargs) -> bool:
+    return max(sizes) if sizes else None
+
+
+def _prune_progress(now: float) -> None:
+    """Dict に残り続ける古い進捗を落とす。"""
+    try:
+        for key, value in list(progress_dict.items()):
+            updated_at = float((value or {}).get("updated_at", 0.0))
+            if now - updated_at > PROGRESS_STALE_SECONDS:
+                progress_dict.pop(key, None)
+    except Exception:  # pragma: no cover - 掃除できなくても実害はない
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)